src/EnergyPlus/ChillerAbsorption.cc

```diff
@@ -1118,7 +1118,7 @@ void BLASTAbsorberSpecs::sizeChiller(EnergyPlusData &state)
     PlantUtilities::RegisterPlantCompDesignFlow(state, this->EvapInletNodeNum, tmpEvapVolFlowRate);
 
     if (PltSizCondNum > 0 && PltSizNum > 0) {
-        if (this->EvapVolFlowRate >= state.dataHVACGlobal->TimeStepSys && tmpNomCap > 0.0) {
+        if (this->EvapVolFlowRate >= HVAC::SmallWaterVolFlow && tmpNomCap > 0.0) {
```

Comment:
Found this one thanks to unit test.

Reply:
oh my gosh.  OK, getting more serious about a units backed floating point representation.

STILL KIDDING ... for now.